src/main/resources/WayProperties_de.properties

```diff
@@ -22,6 +22,7 @@ name.ramp=Auffahrrampe
 name.link=Verbindung
```

Comment:
One last request: please add a default translation for English. Then I'm happy to approve.